compiler_opt/rl/corpus.py

```diff
@@ -224,6 +224,40 @@ def __call__(self, k: int, n: int = 10) -> List[ModuleSpec]:
     return list(results)
 
 
+class WholeCorpusSampler(Sampler):
+  """Returns the entire corpus every time a sample is requested."""
```

Comment:
is the the entire corpus, or it's that subset of IR modules that cover a function graph?

Reply:
It is the subset of IR modules that cover a trace. That is in effect a whole corpus though given the tooling creates a new `corpus_description.json` for the corpus subset.